src/conf/config.rs

```diff
@@ -18,41 +18,52 @@ use {
 /// override previously set values when applied to settings.
 #[derive(Debug, Clone, Deserialize, JsonSchema)]
 pub struct Config {
+    /// Extra arguments appended when a job runs a cargo alias.
     pub additional_alias_args: Option<Vec<String>>,
 
+    /// Name of the job to run when no job was requested explicitly.
     pub default_job: Option<ConcreteJobRef>,
 
     /// Default config for a job
     #[serde(flatten)]
     pub all_jobs: Job,
 
-    /// locations export
+    /// Deprecated single export configuration; use `exports.locations` instead.
     #[deprecated(since = "2.22.0", note = "use exports.locations")]
     pub export: Option<ExportConfig>,
 
+    /// Deprecated toggle for the legacy locations export; use `exports.locations.auto`.
     #[deprecated(since = "2.9.0", note = "use exports.locations.auto")]
     pub export_locations: Option<bool>,
 
+    /// Export configurations keyed by their name.
     #[serde(default)]
     pub exports: HashMap<String, ExportConfig>,
 
+    /// Whether to display the contextual help line in the UI.
     pub help_line: Option<bool>,
 
+    /// Job definitions keyed by their identifier.
     #[serde(default)]
     pub jobs: HashMap<String, Job>,
 
+    /// Custom keybindings layered on top of the defaults.
     pub keybindings: Option<KeyBindings>,
 
+    /// Whether to display the mission output in reverse order.
     pub reverse: Option<bool>,
 
+    /// Whether to show the condensed summary area by default.
     pub summary: Option<bool>,
 
+    /// Deprecated toggle that enables a built-in set of Vim-style keybindings. Use `keybindings` instead.
```

Comment:
While the deprecated tag causes `"deprecated": true` to be added to the schema, not all places understand this, and the description (use keybindings) doesn't flow through to the UI, so it's worth keeping this info here regardless.